pipeline/README.md

```diff
@@ -30,10 +30,35 @@ From command line, it can be done with this script:
 
 knative eventing is used to restart all downstream tasks.  Code rebuilds will trigger image rebuilds.  Image rebuilds will trigger helm upgrades.
 
+## Scoping the deployment
+In the following section the *system* namespace is the namespace specified when running the bootstrap script.
+
+By default the bootstrap script will deploy Fybrik namespaced scoped. Deploying as namespaced scoped allows multiple instances of Fybrik to be installed in unique namespaces in the same cluster.  You can change this behavior but you must do so with care.
+
+### Environment variables
+| Name  | Default  | Valid Values | Description |
+|-------|----------|--------------|-------------|
+| cluster_scoped | false | true, false | Indicates if deploy as cluster scoped meaning the controller will watch and process FybrikApplications from all namespaces. For caveats see [tips](#tips) |
+| use_application_namespace | false | true, false | Specifies if you want the bootstrap script to create a namespace for FybrikApplications. This will be ignore if cluster_scoped=true. If set to false the *system* namespace will be used. |
+
+### Tips
+1. When deploying a Fybrik instance as cluster scoped - **Disclaimer: Only use cluster scope if you know what you’re doing.**
+   1. There can only be one instance of Fybrik deployed in cluster. 
+   2. The controller will be setup so it watches all namespaces for FybrikApplications.
+   3. You cannot run namespaced scoped and cluster scoped in same cluster.
+
+2. When deploying a Fybrik instance as namespaced scoped:
+   1. You can only create FybrikApplication custom resources from one namespace. The default will be to allow only creating from the *system* namespace. You can override this behavior by setting the use_application_namespace environment variable to true, the bootstrap code will create the *system*-app namespace and necessary objects to allow creating FybrikApplications. Hint: You will see the application namespace in the -p mesh-for-data-values= in the tkn command that is returned from running the bootstrap script.
+
+3. When using wkc-connector, the wkc-credentials secret must be in the same namespace you are using when creating FybrikApplication custom resources. The bootstrap script will unconditionally create a wkc-credentials secret in *system* namespace. 
```

Comment:
@laurieaw63 @roee88 Should this be in the open source?

Reply:
> @laurieaw63 @roee88 Should this be in the open source?

That's a question to @ronenkat

Reply:
@ronenkat So are we talking only about #3. Or number 2 as well?

Reply:
Only removed #3.